renderer/components/Onboarding/Steps/Recover.js

```diff
@@ -40,32 +154,94 @@ class SeedWord extends React.Component {
   }
 
   render() {
-    const { index, intl, onPaste, word } = this.props
+    const {
+      bip39WordMap,
+      formState,
+      index,
+      intl,
+      onPaste,
+      setFormApiFieldValue,
+      theme,
+      word,
+    } = this.props
+    const { value } = this.state
+
     return (
-      <Flex key={`word${index}`} alignItems="center" as="li" justifyContent="flex-start" my={1}>
-        <Label htmlFor={`word${index}`} mb={0} width={35}>
-          {index}
-        </Label>
-        <Input
-          field={`word${index}`}
-          hasMessage={false}
-          initialValue={word}
-          mask={this.mask}
-          onPaste={onPaste}
-          placeholder={intl.formatMessage({ ...messages.word_placeholder })}
-          validate={this.validate}
-          validateOnChange
-          variant="thin"
-          willAutoFocus={index === 1}
-        />
-      </Flex>
+      <Downshift
+        onSelect={selection => {
+          this.setState({
+            value: selection,
+          })
+          setFormApiFieldValue(selection)
+        }}
+      >
+        {({
+          getInputProps,
+          getItemProps,
+          getMenuProps,
+          getRootProps,
+          isOpen,
+          inputValue,
+          highlightedIndex,
+        }) => (
+          <Box sx={{ position: 'relative' }} {...getRootProps()}>
+            <Flex
+              key={`word${index}`}
+              alignItems="center"
+              as="li"
+              justifyContent="flex-start"
+              my={1}
+            >
+              <Label htmlFor={`word${index}`} mb={0} width={35}>
+                {index}
+              </Label>
+              <Input
+                {...getInputProps({
+                  field: `word${index}`,
+                  hasMessage: false,
+                  highlightOnValid: true,
+                  initialValue: word,
+                  mask: this.mask,
+                  onChange: this.onInputChange,
+                  onPaste,
+                  placeholder: intl.formatMessage({ ...messages.word_placeholder }),
+                  validate: this.validate,
+                  validateOnChange: true,
+                  value,
+                  variant: 'thin',
+                  willAutoFocus: index === 1,
+                  fieldState: {
```

Comment:
I thought I needed this but now I removed it and nothing breaks or changes at all as far as I can tell. If it turns out this is really needed it can always come back.

Reply:
Done.